### ISSUE TITLE
[OMP] Null value in column setting_type during Upgrade::migrateSubmissionCoverImages

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#6216]
See https://forum.pkp.sfu.ca/t/omp-upgrade-failed-3-2-1-3-2-1-1/62356/9:
```
$ php tools/upgrade.php upgrade
[pre-install]
[load: upgrade.xml]
[version: 3.2.1.1]
[code: Installer Installer::checkPhpVersion]
[data: dbscripts/xml/upgrade/3.1.2_preupdate_user_author_names.xml (skipped)]
[data: dbscripts/xml/upgrade/3.1.2_preupdate_chapters.xml (skipped)]
[data: dbscripts/xml/upgrade/3.1.2_preupdate_categories.xml (skipped)]
[data: dbscripts/xml/upgrade/3.1.2_preupdate_pub_id.xml (skipped)]
[code: Installer Installer::migrateSubmissionCoverImages]
PHP Notice:  Undefined offset: 1 in /home/omp/omp/classes/install/Upgrade.inc.php on line 760
PHP Fatal error:  Uncaught Exception: DB Error: ERROR:  null value in column "setting_type" violates not-null constraint
DETAIL:  Failing row contains (1, en_US, coverImage, a:3:{s:10:"uploadName";s:29:"submission_1_1_coverImage.png";s:12..., null). Query: INSERT INTO submission_settings (submission_id, setting_name, setting_value, locale)
						VALUES (?, ?, ?, ?) in /home/omp/omp/lib/pkp/classes/db/DAO.inc.php:703
Stack trace:
#0 /home/omp/omp/lib/pkp/classes/db/DAO.inc.php(231): DAO->handleError(Object(ADODB_postgres8), 'INSERT INTO sub...')
#1 /home/omp/omp/classes/install/Upgrade.inc.php(814): DAO->update('INSERT INTO sub...', Array)
#2 /home/omp/omp/lib/pkp/classes/install/Installer.inc.php(417): Upgrade->migrateSubmissionCoverImages(Object(Upgrade), Array)
#3 /home/omp/omp/lib/pkp/classes/install/Installer.inc.php(263): Installer->executeAction(Array)
#4 /home/omp/omp/lib/pkp/classes/install/Installer.inc.php(186): Installer->executeInstaller()
#5 /home/omp/omp/lib/pkp/classes/cliTool/UpgradeTool.inc.php(88): Installer- in /home/omp/omp/lib/pkp/classes/db/DAO.inc.php on line 703
```

**Comment by asmecher** [comment]
This error can be resolved in OJS 3.2.1-0, 3.2.1-1, and 3.2.1-2 by applying https://github.com/pkp/omp/commit/22283518680d041be08c8f77632041efe813cdde before running the upgrade script. It appears to affect only PostgreSQL users.

**Comment by asmecher** [comment]
(Committed to `stable-3_2_1` and `master` for OMP.)
